### ISSUE TITLE
Admin Dashboard accessibility audit

### ISSUE THREAD

**Issue by jillpe** [notch8/utk-hyku#485]
# Summary
ref. #41 

The admin dashboard failed the accessibility audit because, at the tested dimensions described below, a website should adjust itself to be viewable vertically. A user should be able to see and use a vertical scrollbar, but not a horizontal one. 

We've determined that this currently does not work because the admin dashboard has a left sidebar that does not collapse by default. Once we manually collapse the sidebar, it passes the audit test. 

Some solution options may include: 

- [ ] When visiting the admin dashboard page, the side bar should be collapsed by default. All users must manually expand the sidebar to view its options. 

  Or, with CSS the above would only happen after certain dimensions are met. (please see the notes section)

For inspiration + things to consider: 

Gitlab passes this requirement! Notice how the left side bar is collapsed, but when you hover over the icons the additional options are revealed. 

![Image](https://user-images.githubusercontent.com/29311858/203172232-d9c3ea42-7b94-4561-80c3-b7ec94913485.png)

Currently, when you hover over the collapsed icons in hyku, the user really has no idea what their significance is. See demo: https://share.getcloudapp.com/xQuw9yKb

# Acceptance Criteria
- [ ] When visiting the admin dashboard page, the side bar should be collapsed by default. All users must manually expand the sidebar to view its options. 
  - [ ]  no horizontal scrolling should be available
- [ ] Viewing the admin dashboard at 320 CSS pixels (see notes section) should pass the accessibility audit. The website should be responsive and align it self for vertical viewing.

# Testing Instructions

Check to see if horizontal scrolling appears when you open and close the sidebar, regardless of the screen resolution size. In particular we confirmed anything above and below 768. 320 for mobile ios gen 4 viewers. 

# Notes

Please reference this document for additional details, section 1.4.10 Reflow: [Report for https___test.hykucommons.org_.pdf](https://github.com/scientist-softserv/utk-hyku/files/10061210/Report.for.https___test.hykucommons.org_.pdf)

**Comment by ShanaLMoore** [comment]
see demo in PR: https://github.com/scientist-softserv/utk-hyku/pull/493

**Comment by jillpe** [comment]
SoftServ QA: Needs rework

<details><summary>320 resolution ❌ </summary>


https://github.com/scientist-softserv/utk-hyku/assets/84697174/1b8ecfc7-d406-4372-9e62-df2c1de41c8f


</details>


<details><summary>1080 resolution ✅ </summary>


https://github.com/scientist-softserv/utk-hyku/assets/84697174/e95ff241-b1af-4ee7-8451-27c9ea9a5add


</details>

<details><summary>780 resolution ✅ </summary>


https://github.com/scientist-softserv/utk-hyku/assets/84697174/832035b2-dc68-4e22-b9a8-313ef4cc5128


</details>

**Comment by jillpe** [comment]
SoftServ QA: ✅ 


https://github.com/scientist-softserv/utk-hyku/assets/84697174/acbea033-3977-4e9a-b72c-081c77d228dc
